### PR TITLE
checked working_with_rgi and plot_mass_balance

### PR DESCRIPTION
**Pull request by lilianschuster**

### CODE REVIEW

File: notebooks/plot_mass_balance.ipynb [hunk 1]
```diff
@@ -333,7 +335,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Similarly, the simulation ends at the last possible date for which we have climate data for (calendar year 2016):"
+    "Similarly, the simulation ends at the last possible date for which we have climate data for (calendar year 2019):"
```
Comment (lilianschuster) [inline notebooks/plot_mass_balance.ipynb:338]: I am not sure about the 2019 here, but I think 2016 is wrong!

File: notebooks/plot_mass_balance.ipynb [hunk 2]
```diff
@@ -388,7 +390,7 @@
    "source": [
     "A few comments:\n",
     "- the systematic difference between WGMS and OGGM is due to the calibration of the model, which is done with fixed geometry (the bias is zero over the entire calibration period but changes sign over the 50 year period, see plot above)\n",
-    "- the difference between OGGM dynamics and fixed geometry is due to:\n",
+    "- the difference between OGGM dynamics and fixed geometry (not visible for this short time period) is due to:\n",
```
Comment (lilianschuster) [inline notebooks/plot_mass_balance.ipynb:393]: if I understand this right, there are no changes in the plot visible between OGGM dynamics and fixed geometry run for this short time period of 15 years, therefore sth. like that could be added to make that clearer!

File: notebooks/plot_mass_balance.ipynb (hunk 1 above)
Comment (fmaussion) [reply notebooks/plot_mass_balance.ipynb:338]: yes! The end date is actually hydro year 2020, month 01 (i.e. Oct 1st 2019).

File: notebooks/plot_mass_balance.ipynb (hunk 2 above)
Comment (fmaussion) [reply notebooks/plot_mass_balance.ipynb:393]: yes - one can see them a very little bit, but not much...

**Comment by fmaussion** [comment]
Thanks a bunch!

**Pull request merged**